mods/bones/init.lua

```diff
@@ -6,8 +6,15 @@
 -- Load support for MT game translation.
 local S = minetest.get_translator("bones")
 
+local bones_max_slots = tonumber(minetest.settings:get("bones_max_slots")) or 15 * 10
```

Comment:
Needs documentation in `minetest.conf.example`.

Reply:
> Needs documentation in `minetest.conf.example`.

you most probably mean the minetest.conf.example inside minetest_game (not in minetest)
however there are two minetest.conf.example files, and they differ

Reply:
fixed in 357f87a